spec/Normalizer/NormalizerSpec.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace spec\Http\Client\Plugin\Normalizer;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\UriInterface;
+use Psr\Log\LoggerInterface;
+
+class NormalizerSpec extends ObjectBehavior
+{
+    function it_is_initializable(LoggerInterface $logger)
+    {
+        $this->beAnInstanceOf('Http\Client\Plugin\Normalizer\Normalizer');
+    }
+
+    function it_normalize_request_to_string(RequestInterface $request)
+    {
+        $request->getMethod()->shouldBeCalled()->willReturn('GET');
```

Comment:
shouldBeCalled is only necessary when the behavior is the fact that a method is called. For example you expect wait to be called on the promise. But in these cases, the behavior is the value being returned, not the fact that the method is called.